lib/rules/selector-type-case/index.js

```diff
@@ -22,6 +22,10 @@ const meta = {
 	fixable: true,
 };
 
+const STARTS_A_TAG_NAME_REGEX = /(?:[^.#[:\w-]|^)\w/;
```

Comment:
[ask] Can we use `[a-zA-Z]` instead of `\w`?

Reply:
yes!

https://html.spec.whatwg.org/multipage/custom-elements.html#valid-custom-element-name